src/webgl/p5.RendererGL.js

```diff
@@ -935,6 +935,15 @@ p5.RendererGL = class RendererGL extends Renderer {
     this.clear(_r, _g, _b, _a);
   }
 
+  // Combines the model and view matrices to get the uMVMatrix
+  // This method will be reusable wherever you need to update the combined matrix.
+  calculateCombinedMatrix() {
+    const modelMatrix = this.uModelMatrix;
```

Comment:
I think this and the line below would need to access `states` now:
```suggestion
    const modelMatrix = this.states.uModelMatrix;
```